conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -466,42 +458,33 @@ public void testBinaryServerSideFailureAfterFewBytesSent() {
     }
 
     @Test
-    @Timeout(20)
+    @Timeout(40)
```

Comment:
I am intentionally cheating here, and I plan to fix this in a FLUP. Lots of back and forth with @carterkozak we do not think this is a bug nor a regression, but actually an interaction between two testing suites. Which I will fix in a FLUP.